lib/is-node-under-scope.js

```diff
@@ -1,11 +1,11 @@
 var isUnderScope = require('./is-under-scope');
```

Comment:
> I actually have to admit that I'm not using this plugin any more due to being able to drop IE11 support.
> 
> It was used in production and I cannot really remember any issues.
> 
> As it's been so long since I made the changes I know exactly as much as you on how and why they were made. : )
>
> https://github.com/MadLittleMods/postcss-css-variables/pull/126#issuecomment-815084707

@siilike No worries, I understand. I might just add the comments to the tests where I pointed it out and merge this.

Seems like a decent improvement regardless of the code and us trying to understand it better.